features/org.eclipse.rap.tools.feature/feature.xml

```diff
@@ -132,12 +132,6 @@
          version="0.0.0"
          unpack="false"/>
 
-   <plugin
```

Comment:
Do we need to add "jakarta.servlet-api" here?

Reply:
Initially I added it, yes, but then I saw that it was already included at the very end of the file for quite some time.